server/utils/db/db_utils.py

```
@@ -1885,84 +1885,57 @@ def initialize_tables():
-                    DO $$
-                    BEGIN
+                # RLS condition: enforce org scoping when session var is set,
+                # allow through when unset (prevents crashes from code paths that
+                # don't call SET myapp.current_org_id before writing)
+                _rls_using = f"""
+                    org_id IS NOT NULL
+                    AND (
+                        COALESCE(current_setting('myapp.current_org_id', true), '') = ''
+                        OR org_id = current_setting('myapp.current_org_id', true)::text
+                    )
+                """
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)